gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -918,8 +918,17 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
     }
     case GTT_STAND:
     {
-      filter->data_stand.mode =
-          gst_tensor_transform_get_stand_mode (filter->option);
+      gchar **strv = NULL;
+
+      strv = g_strsplit (filter->option, ":", -1);
```

Comment:
Q: Is it possible that `filter->option` is a empty string? Otherwise, need to check this first.

Reply:
line 740 checks this case.

Reply:
I meant a empty string `""` not nullptr.

Reply:
then gst_tensor_transform_get_stand_mode() will return STAND_END, which means error case.

Reply:
I think `strv[0]` can be a problem. 
from the GLib manual
```
As a special case, the result of splitting the empty string "" is an empty vector, not a vector containing a single string. The reason for this special case is that being able to represent an empty vector is typically more useful than consistent handling of empty elements. If you do need to represent empty elements, you'll need to check for the empty string before calling g_strsplit().
```

Reply:
I will make another PR to handle option string is empty string.

Reply:
Got it